### DISCUSSION TITLE
Feature Request: Cascade windows while maintaining sizes or adjust sizes of cascaded windows.

### DISCUSSION THREAD

**Discussion by bigplayer-ai** [rxhanson/RectanglePro-Community#471] [Ideas]
# Request: Cascade windows while maintaining sizes or adjust sizes of cascaded windows.

Is there a way to cascade windows while maintaining their sizes or adjust the sizes of cascaded windows?

**Comment by t4k** [comment]
Came here to ask about this.

**Comment by rxhanson** [comment]
Looks like this one fell off my desk. Sorry!

You can specify width and height with the following Terminal commands:

```
defaults write com.knollsoft.Hookshot cascadeWidth -int 500
defaults write com.knollsoft.Hookshot cascadeHeight -int 500
```

Replace 500 with your desired "pixel" size.

Restart the app after executing.

I'll consider adding an option to maintain size in a future release.
